src/massager.ts

```diff
@@ -96,7 +96,9 @@ export function replan(inPaths: (Vec2[] & { stroke?: string, groupId?: string })
     },
     penDropDuration: planOptions.penDropDuration,
     penLiftDuration: planOptions.penLiftDuration,
-  });
+  },
+  planOptions.origin,
```

Comment:
I think here is where we should convert before recalculating the plan:

```suggestion
  planOptions.origin * device.stepsPerMm,
```

Reply:
Oh shoot, yeah, of course.